src/ldw-barsPlus.js

```diff
@@ -259,17 +260,18 @@ export default {
         : (p.indexOf(a.key) > p.indexOf(b.key) ? 1 : 0));
     });
     n.forEach(function (d, idx) {
-      var t = 0, v = [], j = 0, num, txt;
+      var t = 0, v = [], j = 0, num, txt, mNum;
```

Comment:
Try to give new variables descriptive names and generally refrain from abbreviations. 

Take the extra minute or two to make the surrounding code a bit more readable, `n.forEach(d` for example, giving those descriptive names would save the next one looking at this a bit of cognitive load and time.

No need to fix everything, just leave it a little bit better than off than you found it.

Reply:
i think it's pretty declaretive to anyone who's developing extensions for Qlik
since mNum = Measure Number
But i get you point